ui-core/src/types/forms.ts

```diff
@@ -14,6 +14,7 @@ export type SurveyType = 'RESEARCH' | 'OUTREACH'
 
 export type Survey = VersionedForm & {
   surveyType: SurveyType
+  blurb?: string
```

Comment:
Should we allow this to be empty?
I forget from the other PR if there is a placeholder if this is not specified.

Reply:
It's a nullable column and it may not exist for research surveys. Since we don't yet have a way for users to set/display blurbs for those research surveys, I've left it optional for now, but that could change as this feature matures.